apps/fm/test/unit/fm_test.js

```diff
@@ -70,6 +115,54 @@ suite('FM', function() {
       assert.equal($('frequency').textContent, 87.5);
     });
 
+    test('compare new set frequency with aria-valuenow', function() {
+      var frequency = 95.7;
+      var freq = frequencyDialer.setFrequency(frequency);
+      assert.equal($('dialer-container').getAttribute('aria-valuenow'),
+        frequency.toString());
+    });
+
+    test('screen reader swipe up on dialer should advance 0.1 Hz', function() {
```

Comment:
I think it's fine to have just one test both up and down and call:
testScreenReaderSwipe('up', 95.7, 95.8);
testScreenReaderSwipe('down', 95.7, 95.6);
in one place